pkg/services/business_case.go

```diff
@@ -209,6 +210,25 @@ func NewUpdateBusinessCase(
 				Operation: apperrors.QuerySave,
 			}
 		}
+		// Right now, add some validations for sending an email here
+		// Similar to system intake,
+		// these should be covered by validations above
+		if businessCase.Status == models.BusinessCaseStatusSUBMITTED &&
+			existingBusinessCase.Status == models.BusinessCaseStatusDRAFT {
+			if !businessCase.Requester.Valid {
+				validationError := apperrors.NewValidationError(
+					errors.New("failed to validate for email"),
+					businessCase,
+					businessCase.ID.String(),
+				)
+				validationError.WithValidation("Requester", "is required")
+				return businessCase, &validationError
```

Comment:
still returning a business case when there's a failure. the client should be able to resume with an error in this case, but we want to log an error